src/library-authoring/generic/status-widget/StatusWidget.scss

```diff
@@ -1,12 +1,26 @@
+%draft-status {
+  background-color: #FDF3E9;
+  border-color: #F4B57B !important;
```

Comment:
Should we add a design token color for these? see e.g https://github.com/openedx/paragon/blob/f154be3332d917db51fdb68bf9d20e891212f69a/styles/css/themes/light/variables.css#L640